applications/search/src/pages/search-page/search-page.jsx

```diff
@@ -243,6 +243,19 @@ export const SearchPage = props => {
             />
           </Route>
         </Switch>
+        <div className="twitter-container">
```

Comment:
I stedet for egen styling klasse for .twitter-container så kan du også bruke de innebygde klassene for flexbox fra bootstrap som her da vil kunne bli: 'd-flex justify-items-end mt-5'